src/main/java/org/corfudb/cmdlets/corfu_layout.java

```diff
@@ -35,6 +38,7 @@
 public class corfu_layout implements ICmdlet {
 
     private static Map<String, NettyClientRouter> routers = new ConcurrentHashMap<>();
+    private static Map<String, LayoutView> layoutViews = new ConcurrentHashMap<>();
```

Comment:
Not sure I understand your response, but if it makes anything clearer, code which requires a CorfuRuntime belongs in the `corfu_layouts` cmdlet, not the `corfu_layout` cmdlet.